Classes/Hooks/Form.php

```diff
@@ -100,7 +100,7 @@ public function afterSubmit(FormRuntime $runtime, $element, $value, $requestArgu
     protected function crFieldHasBeenVerified(FormRuntime $runtime): bool
     {
         return $runtime->getFormState() &&
-            $runtime->getFormState()->getFormValue(self::FIELD_ID) === sha1($this->getHmacSalt($runtime));
+            hash_equals($runtime->getFormState()->getFormValue(self::FIELD_ID), sha1($this->getHmacSalt($runtime)));
```

Comment:
we need to verify, that the first and second value of the comparision are not null.